src/App.tsx

```diff
@@ -10,33 +11,20 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [selectedTabId, setselectedTabId] = React.useState(tabs[0]);
```

Comment:
just import useState
```suggestion
  const [selectedTabId, setselectedTabId] = useState(tabs[0]);
```